Assets/Scripts/Loop/LoopController.cs

```diff
@@ -12,14 +12,14 @@
 
 public class LoopController : MonoBehaviour
 {
-    private Timer _loopTimer;
-    [SerializeField] private string _loopTimerName;
-    [SerializeField] private int _loopTimerTime;
-    [SerializeField] private int _endScreenDelay;
-    [SerializeField] private NpcEvent _temporaryLoop;
-    [SerializeField] private NpcEventTags _temporaryTag;
-    public int LoopTimerTimer => _loopTimerTime;
-    public string LoopTimerName => _loopTimerName;
+    //private Timer _loopTimer;
```

Comment:
This applies to all of the scripts you edited, but if all these commented out lines aren't needed, they could be removed to clean up the code a bit.